sympy/printing/tests/test_smtlib.py

```diff
@@ -13,30 +14,41 @@
 
 
 def test_Integer():
+    warning_filters = list(warnings.filters)
+    warnings.simplefilter('ignore')
     assert smtlib_code(Integer(67)) == "67"
     assert smtlib_code(Integer(-1)) == "-1"
+    warnings.filters = warning_filters
```

Comment:
This kind of thing would need to be guarded with `try/finally` but in any case there is the `ignore_warnings` context manager for this:
https://github.com/sympy/sympy/blob/e8c22f6eac7314be8d92590bfff92ced79ee03e2/sympy/functions/elementary/tests/test_miscellaneous.py#L349-L350

Reply:
We can make a top-level helper in the test file for this that runs the smtlib_code function and catches the warnings, so you don't have to do this for every single test (maybe even have the function assert that certain warnings are emitted).

Reply:
fixed!